### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by srossross**
part of https://github.com/Plantiga/plantiga.io/issues/3032

### CODE REVIEW

**Comment by srossross** [comment]
hey guys, what else needs to be documented here for a new dev contributing to plantiga?

**Review by dsludwig — APPROVED** [review]

**Review by umarqshi — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## We Use [Github Flow](https://guides.github.com/introduction/flow/index.html), So All Code Changes Happen Through Pull Requests
+Pull requests are the best way to propose changes to the codebase (we use [Github Flow](https://guides.github.com/introduction/flow/index.html)). We actively welcome your pull requests:
+
+1. Fork the repo and create your branch from `master`.
```
Comment (ahollist) [inline CONTRIBUTING.md:4]: Is this for internal developers or external? If it's for internal, should they be forking or cloning?

Comment (srossross) [reply CONTRIBUTING.md:4]: Good catch - I think they should be cloning, otherwise they would not get test automation from CircleCI CI/CD

Comment (srossross) [reply CONTRIBUTING.md]: updated

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+Pull requests are the best way to propose changes to the codebase (we use [Github Flow](https://guides.github.com/introduction/flow/index.html)). We actively welcome your pull requests:
+
+1. Clone the repo and create your branch from `master`.
+2. If you've added code that should be tested, add tests.
+3. If you've changed APIs, update the documentation.
+4. Ensure the test suite passes.
+5. Make sure your code lints.
+6. Make sure you update the README
+7. Issue that pull request!
+8. If it requires manual tests after the merge, add a `## testing` section and descripe what the person on release duties should do
```
Comment (ahollist) [inline CONTRIBUTING.md:11]:
```suggestion
8. If it requires manual tests after the merge, add a `## testing` section and describe what the person on release duties should do
```

**Review by ahollist — APPROVED** [review]

**Review by Rowansdabomb — APPROVED** [review]

**Pull request merged**